src/App.js

```diff
@@ -1,15 +1,49 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link,
+} from 'react-router-dom';
 import Calculate from './components/Calculator';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Let's keep our codebase clean. Kindly remove unnecessary disabling of linters here. Linters return no errors when this is removed and the app works just fine.